frontend/src/pages/login.tsx

```diff
@@ -83,6 +103,11 @@ export default function LoginDesktop() {
           }}>
           Log In
```

Comment:
Use condition rendering here, something like:
` {loading ? <CircularProgress size={24} color="inherit" /> : 'Log In'}`